packages/manager/modules/pci/src/projects/project/additional-ips/order/order.controller.js

```diff
@@ -331,4 +340,50 @@ export default class AdditionalIpController {
       ] || GUIDE_URLS.REGIONS_AVAILABILITY.DEFAULT
     );
   }
+
+  getDefaultValue(field) {
+    return (
+      this.defaults[field.name] !== undefined &&
+      isFunction(field.setDefault) &&
+      field.setDefault(this.defaults[field.name], this[field.availableOptions])
+    );
+  }
+
+  onChangeAfterDefaultValueSet(methodToCall, params) {
+    if (this[methodToCall] && isFunction(this[methodToCall])) {
+      return this[methodToCall].call(
+        this,
+        ...params.map((prop) => get(this, prop)),
+      );
+    }
+    return null;
+  }
+
+  async setDefaultSelections() {
+    this.loadingDefaultValues = true;
+    for (let i = 0; i < DEFAULTS_MODEL.length; i += 1) {
+      const step = DEFAULTS_MODEL[i];
+      for (let j = 0; j < step.fields.length; j += 1) {
+        const defaultvalue = this.getDefaultValue(step.fields[j]);
+        if (defaultvalue !== undefined) {
```

Comment:
suggestion:  if(defaultvalue), this will check for undefined and null both.

Reply:
Sometimes, `null` can be a valid value. So intentionally checking only for `undefined`.